Joomla/Tests/ControlStructures/ControlStructuresBracketsUnitTest.inc

```diff
@@ -262,7 +262,7 @@ class Application
 					if ($thing)
 						{
 						bar();
-				}
+					}
```

Comment:
Seems like this line is thinking it belongs to the closure function and not the control structure. I get why you chose to re-indent it...
(I can't remember off the top of my head if this sniff can even check for the alignment of the second bracket or if it is strictly just looking for the brace is on a new line and the alignment of that brace. I think it might be just the first bracket)

Reply:
It's just the first bracket when i inspected further. That's why i reindented it :)